### PR TITLE
Update to 1.0.2

### PR DESCRIPTION
**Pull request by philippjfr**
Update to latest along with dependencies.

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -36,19 +36,20 @@ requirements:
   run:
     - python
     - bleach
-    - bokeh >=2.4.0,<2.5.0
+    - packaging
+    - bokeh >=3.1.1,<3.2
     - markdown
     - param >=1.12.0
     - pyct >=0.4.4
```
Comment (skupr-anaconda) [inline recipe/meta.yaml:43]: `pyct` isn't mentioned in `install_requires` https://github.com/holoviz/panel/blob/15acb1b148cf1a6ea39bf19670dc281d1bac7f03/setup.py#L103-L117
Is it still required?

**Review by boldorider4 — COMMENTED** [review]
Was `recipe/index-ts-fix.patch` written by you or did you take it from a GH issue/PR?
I also don't see this patch being reference in the `meta.yaml`.

**Review by skupr-anaconda — APPROVED** [review]
LGTM

File: recipe/meta.yaml (hunk 1 above)
Comment (philippjfr) [reply recipe/meta.yaml]: It is not, good catch!

**Review by ELundby45 — APPROVED** [review]

**Pull request merged**